JamfUploaderProcessors/READMEs/JamfPackageUploader.md

```diff
@@ -72,13 +72,13 @@ Can be run as a post-processor for a pkg recipe or in a child recipe. The parent
   - **description:** Password of api user, optionally set as a key in the com.github.autopkg preference file.
 - **SMB_URL:**
   - **required:** False
-  - **description:** URL to a Jamf Pro fileshare distribution point which should be in the form `smb://server/share`.
+  - **description:** URL to a Jamf Pro fileshare distribution point which should be in the form `smb://server/share`. If you have multiple SMB shares you wish to mount, you can separate those urls with ';;' (ex. `smb://xsrv1.example.com/CasperShare;;smb://xsrv2.example.com/Caspershare`) and the uploader will iterate over that list.
 - **SMB_USERNAME:**
   - **required:** False
-  - **description:** Username of account with appropriate access to jss, optionally set as a key in the com.github.autopkg preference file.
+  - **description:** Username of account with appropriate access to jss, optionally set as a key in the com.github.autopkg preference file. If you have multiple SMB shares you wish to mount that have different usernames, you can separate those usernames with ';;' (ex. `xsrvupload1;;xsrvupload2`) in the corresponding order to your SMB_URL list, otherwise the same account will be used for all SMB endpoints.
 - **SMB_PASSWORD:**
   - **required:** False
-  - **description:** Password of api user, optionally set as a key in the com.github.autopkg preference file.
+  - **description:** Password of api user, optionally set as a key in the com.github.autopkg preference file. If you have multiple SMB shares you wish to mount that have different passwords, you can separate those passwords with ';;' (ex. `xsrvpw1;;xsrvpw2`) in the corresponding order to your SMB_URL list, otherwise the same account will be used for all SMB endpoints.
```

Comment:
What happens if the password contains semi-colons?

Reply:
That's why I used two semi-colons as the delimiter, but realistically, I can't think of a good delimiter that couldn't potentially occur 'naturally', or cause issues (if, for instance, a password ends with a semi-colon, leading to the script picking up the contents too soon). Maybe a heads up check to see if three or more semi-colons occur within a string, only pick up the index of the 1st to last instance for the index (which wouldn't affect counting the number of shares, since that's determined using only the URLs). It would also be pretty easy to change it to something like else that's a bit more unlikely to be encountered.